test/odbc/config.txt

```diff
@@ -1,6 +1,15 @@
-ODBC_DRIVER_NAME=
-BABEL_DB_SERVER=
-BABEL_DB_PORT=
-BABEL_DB_USER=
-BABEL_DB_PASSWORD=
-BABEL_DB_NAME=master
\ No newline at end of file
+MSSQL_ODBC_DRIVER_NAME=ODBC_Driver_17
+MSSQL_BABEL_DB_SERVER=localhost
+MSSQL_BABEL_DB_PORT=1433
+MSSQL_BABEL_DB_USER=jdbc_user
+MSSQL_BABEL_DB_PASSWORD=12345678
+MSSQL_BABEL_DB_NAME=master
+MSSQL_TEST_TO_RUN=MSSQL
+
+PSQL_ODBC_DRIVER_NAME=ODBC_Driver_17_PostgreSQL
+PSQL_BABEL_DB_SERVER=localhost
+PSQL_BABEL_DB_PORT=5432
+PSQL_BABEL_DB_USER=jdbc_user
+PSQL_BABEL_DB_PASSWORD=12345678
+PSQL_BABEL_DB_NAME=jdbc_testdb
+PSQL_TEST_TO_RUN=PSQL
```

Comment:
Are the `_TEST_TO_RUN` fields necessary? (assuming that we're removing the constants for default valuesI I would assume if MSSQL fields were filled out, I would run the MSSQL test suite and if the PSQL test were filled out, I would run the PSQL test suite.

Reply:
I think this field is useful here. Else, in the main.cpp file we would do a bunch of `if` statements such as:
```
string mssql_tests = "MSSQL"
string psql_tests = "PSQL"

if MSSQL fields exist:
    add mssql_tests to list of tests to run
if PSSQL fields exist:
    add psql_tests to list of tests to run
...
```
Also, if we were to change the `_TEST_TO_RUN` field in the config.txt this would not require the odbc test framework to be rebuilt.

Reply:
Hmm, currently main.cpp handles the filtering by the user-input and not driver availability I'm wondering if it should be driver availability. 

Just a thought though (no need to change if you feel against).  If we were to make that new class that parses through the config file and environment variable, that could probably handle it. Create a method like `DoesMssqlDriverExist()`  (or `IsValidMssqlConfig()))` or of similar name that returns true if all fields are filled out) and if it returns true, add mssql to the list of tests. And the same with pssql. 

We could still avoid having to rebuild the odbc test framework by using the map constants in constants.cpp. 
```
if IsValidMssqlConfig():
     filter_string.append(server_to_odbc_types[MSSQL]);
     filter_string.append("*:");
if IsValidPsqlConfig():
     filter_string.append(server_to_odbc_types[PSSQL]);
... (could do && condition for hybrid)
```

Again just a thought though, not sure about the limitations and this might work.